assignment/api/src/java/org/sakaiproject/assignment/api/AddAssignmentBullhornHandler.java

```diff
@@ -150,8 +146,6 @@ protected void doInTransactionWithoutResult(TransactionStatus status) {
             }
         });
 
-        users.forEach(u -> countCache.remove(u));
```

Comment:
what's the logic around removing this cache?

Reply:
We don't maintain counts server side any longer. The clientside stuff just counts them as they come in.

Reply:
In the entityprovider, there used to be a call just to get the counts - that was what got short polled every 5 seconds. That call is no longer happening, so no cache needed.